### ISSUE TITLE
move sqlite database in docker image to separate directory

### ISSUE THREAD

**Issue by hojerst** [shlinkio/shlink#1724]
#### Summary

Hi,

I'm running a very small, personal shlink instance for some links. For my setup the sqlite setup would be completely sufficient and a external database is just a complete overkill. However for the sake of easy deployment, I want to run everything inside docker. I'm using the official docker image at `shlinkio/shlink` on dockerhub for that. However, the storage location for the sqlite database makes it quite hard to store the database in a persistent volume / mount. Currently the sqllite database is in `data/database.sqlite`. Unfortunately its unnecessarily hard to mount a volume for a single file as there are other files need in this folder.

TL;DR: Would it be possible to either change the default location of the sqllite database to something like `data/db/database.sqlite` (so db file is in an otherwise empty folder) or provide an option to change the storage location (e.g. through an environment variable)?

This would greatly simplify Docker/Kubernetes deployments using sqllite

**Comment by acelaya** [comment]
See https://github.com/shlinkio/shlink/pull/1522
